apps/bare-expo/android/app/build.gradle

```diff
@@ -197,7 +197,7 @@ dependencies {
 class ExecuteSetupAndroidProject implements Plugin<Project> {
     @Override
     void apply(Project target) {
-        target.providers.exec {
+        target.exec {
```

Comment:
`target.providers.exec()` has been laziness and has not been executed from recent gradle upgrade.